java/core/src/test/org/apache/orc/TestOrcTimezonePPD.java

```diff
@@ -294,7 +294,7 @@ public void testTimestampMinMaxAndBloomFilter() throws Exception {
         .bufferSize(10000).bloomFilterColumns("ts"));
     assertEquals(writerTimeZone, TimeZone.getDefault().getID());
     List<String> ts = Lists.newArrayList();
-    ts.add("2007-08-01 00:00:00.0");
+    ts.add("2007-08-01 01:00:00.0");
```

Comment:
What is the reason behind this 1 hour change? It doesn't seem relevant to sub-millisec issues.